### ISSUE TITLE
[pagination] [accessibility WCAG section508] - pagination 

### ISSUE THREAD

**Issue by hugo-valcourt** [element-plus/element-plus#11730]
Bug Type: **`Other`**

## Environment

- Vue Version: `3.2.47`
- Element Plus Version: `2.2.32`
- Browser / OS: `110.0.5481.177 (arm64) / MacOs`
- Build Tool: `Vite`

### Related Component

- `el-pagination`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/pagination.html)

### Steps to reproduce

With a WCAG validator.

- In chrome, dev tools, lighthouse, you can check the Accessibility
- In storybook with the Accessibility Tab

### What is Expected?

Should not give accessibility error or warning

### What is actually happening?

Elements must only use allowed ARIA attributes

Ensures ARIA attributes are allowed for an element's role

https://dequeuniversity.com/rules/axe/4.6/aria-allowed-attr?application=axeAPI
.el-pagination

Serious
aria-label attribute is not well supported with role "null".
cat.aria wcag2a wcag412

*****

ARIA roles used must conform to valid values

Ensures all elements with a role attribute use a valid value

https://dequeuniversity.com/rules/axe/4.6/aria-roles?application=axeAPI
.el-pagination

Critical
Role must be one of the valid ARIA roles: pagination
cat.aria wcag2a wcag412

*****

Buttons must have discernible text

Ensures buttons have discernible text

https://dequeuniversity.com/rules/axe/4.6/aria-roles?application=axeAPI
.btn-prev

Critical
Element does not have inner text that is visible to screen readers

Serious
aria-label attribute does not exist or is empty

Serious
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Serious
Element has no title attribute

Minor
Element's default semantics were not overridden with role="none" or role="presentation"
.btn-next

Critical
Element does not have inner text that is visible to screen readers

Serious
aria-label attribute does not exist or is empty

Serious
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Serious
Element has no title attribute

Minor
Element's default semantics were not overridden with role="none" or role="presentation"
cat.name-role-value

wcag2a wcag412 section508 section508.22.a ACT

******

Buttons do not have an accessible name
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers. https://dequeuniversity.com/rules/axe/4.4/button-name?utm_source=lighthouse&utm_medium=devtools
Failing Elements

button.btn-prev.is-first
button.btn-next.is-last

### Additional comments

Should always pass wcag and section508 accessibility. You could provide accessibility default in the component. And maybe add some properties to override them> In some juridictions, the law force us to follow accessibility rules, so it's very important.

Thanks for the good work
